pkg/distro/defs/rhel-10/distro.yaml

```diff
@@ -571,7 +571,8 @@ image_types:
       <<: *default_partition_tables
     package_sets:
       os:
-        - include:
+        - &qcow2_pkgset
+          include:
```

Comment:
This change does not seem to be needed, because `qcow2_pkgset` is not used anywhere AFAICT.

Reply:
Thanks! That was a leftover from a previous approach, I've removed it.